File: 프로그래머스/2/64065. 튜플/튜플.js
function solution(s) {
    const numbers = s.match(/\d+/g);
    
    const count = {};
    numbers.forEach((num) => {
        count[num] = (count[num] || 0) + 1;
    });
    
    const sorted = Object.entries(count).sort((a, b) => b[1] - a[1]);
    return sorted.map(([num]) => +num)
}

File: 프로그래머스/2/64065. 튜플/튜플.py
import re
from collections import Counter
def solution(s):
    numbers = re.findall(r'\d+', s)
    
    count = Counter(numbers)
    sorted_nums = sorted(count.items(), key = lambda x: -x[1])
    
    return [int(num) for num, _ in sorted_nums]
    